### PR TITLE
docs(tutorials): add new VDP 101 tutorials

### LINKED ISSUE #453: update vdp 101 with the new pipeline inputs structure

### PR DESCRIPTION
**Pull request by bryan107**
Because

- VDP 101 was created to provider new users a step-by-step tutorials. The aim is to ensure inexperience users including students can easily get start with these tutorials.  

This commit

- 6 new VDP 101 tutorial documents 
- still need to check how to organise it with the original tutorials and make sure all the links works (there are cross-reference links between new tutorials yet have public links).
- refactor old tutorials

Close #453

### CODE REVIEW

File: tutorials/vdp-101-1-introduction.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+draft: false
+description: "*Versatile Data Pipeline (VDP) is an open-source unstructured data Extract-Transform-Load (ETL) tool to streamline the end-to-end unstructured data processing pipeline with Artificial Intelligence (AI)."
+aiTask: "objectDetection"
+sourceConnector: "HTTP"
+destinationConnector: "HTTP"
+publishedOn: "2023-01-30T14:26:00"
+placeholderColor: "bg-instillYellow50"
+themeImgSrc: "/tutorial-assets/vdp-101/cover-animation.gif"
+themeImgThumbnailSrc: "/tutorial-assets/vdp-101/cover-static.png"
+useCase: "Prototyping"
```
Comment (xiaofei-du) [inline tutorials/vdp-101-1-introduction.mdx:13]: Can use a new use case type for all the VDP 101 tutorials.

**Review by xiaofei-du — CHANGES_REQUESTED** [review]
Wait for the following issues to be resolved
- https://github.com/instill-ai/instill.tech/issues/452 
- https://github.com/instill-ai/instill.tech/issues/455

**Comment by bryan107** [comment]
Waiting for the following issues to be resolved

- #461 

This PR will also be released with the example PR https://github.com/instill-ai/vdp/pull/187

File: tutorials/vdp-101-1-introduction.mdx (hunk 1 above)
Comment (bryan107) [reply tutorials/vdp-101-1-introduction.mdx]: Both issues were resolved after rebase.

**Pull request merged**